### PR TITLE
adding description parameter to bigip_monitor_dns

### LINKED ISSUE #833: Add description parameter to bigip_monitor_dns
Add description parameter to bigip_monitor_dns

spawns from: #827

### PR DESCRIPTION
**Pull request by wojtek0806**
fixes #833

### CODE REVIEW

File: test/integration/targets/bigip_monitor_dns/tasks/teardown.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+- name: Remove DNS Monitor
```
Comment (caphrim007) [inline test/integration/targets/bigip_monitor_dns/tasks/teardown.yaml]: this work should be done in the issue-00833.yaml file. teardown is used for tearing down things that are not part of the stuff being tested (tearing down dependencies)

File: library/modules/bigip_monitor_dns.py [hunk 2]
```diff
@@ -924,14 +932,14 @@ def remove_from_device(self):
             self.client.provider['server_port'],
             transform_name(self.want.partition, self.want.name)
         )
-        resp = self.client.api.delete(uri)
-        if resp.status == 200:
+        response = self.client.api.delete(uri)
```
[the remark this answers is not in the document]
Comment (wojtek0806) [reply library/modules/bigip_monitor_dns.py]: why are they showing as invalid in the IDE? I am curious :)

**Comment by wojtek0806** [comment]
@caphrim007 implemented changes

File: test/integration/targets/bigip_monitor_dns/tasks/issue-00833.yaml [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+- name: Issue 00833 - Create DNS Monitor with Description - Idempotent check
+  bigip_monitor_dns:
+    name: "{{ monitor_name }}"
+    query_name: "{{ query_name }}"
+    description: "{{ description }}"
+    state: present
+  register: result
+
+- name: Assert Issue 00833 - Create DNS Monitor with Description - Idempotent check
```
Comment (caphrim007) [inline test/integration/targets/bigip_monitor_dns/tasks/issue-00833.yaml]: Issue 00833 goes *before* the "Assert" string.

File: test/integration/targets/bigip_monitor_dns/tasks/issue-00833.yaml [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+- name: Issue 00833 - Modify DNS Monitor Description
+  bigip_monitor_dns:
+    name: "{{ monitor_name }}"
+    query_name: "{{ query_name }}"
+    description: "{{ description_2 }}"
+    state: present
+  register: result
+
+- name: Assert Issue 00833 - Modify DNS Monitor Description
```
Comment (caphrim007) [inline test/integration/targets/bigip_monitor_dns/tasks/issue-00833.yaml]: Issue before Assert

File: test/integration/targets/bigip_monitor_dns/tasks/issue-00833.yaml [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+
+- name: Modify Issue 00833 - Modify DNS Monitor Description - Idempotent check
+  bigip_monitor_dns:
+    name: "{{ monitor_name }}"
+    query_name: "{{ query_name }}"
+    description: "{{ description_2 }}"
+    state: present
+  register: result
+
+- name: Assert Issue 00833 - Modify DNS Monitor Description - Idempotent check
```
Comment (caphrim007) [inline test/integration/targets/bigip_monitor_dns/tasks/issue-00833.yaml]: Issue before Assert

**Pull request merged**